src/components/homepage/filter/FilterPrice.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { handlerFilterByPrice } from '../../../features/FilterSlice';
+import Notification from '../../alert';
+
+const FilterPrice = (props) => {
+
+    const [price_gte, setprice_gte] = useState();
```

Comment:
```suggestion
    const [priceGte, setPriceGte] = useState();
```
Chỉnh camelCase cho tất cả các hàm setState